docs/src/pages/[platform]/connected-components/account-settings/account-settings.react.mdx

```diff
@@ -12,6 +14,8 @@ Account Settings components are a set of standalone components that add [user ma
 
 ## Quick Start
 
+<Fragment>{() => import('../shared/AppDirectoryAlert.mdx')}</Fragment>
```

Comment:
Nittish: Could this just be a named component to simplify a bit more?

Reply:
Think we already have some, like the [`ReactPropsTable`](https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/components/propsTable/ReactPropsTable.tsx)